app/assets/stylesheets/components/buttons.scss

```diff
@@ -132,7 +132,9 @@
 }
 
 .js-focus-visible .crayons-btn.focus-visible:focus,
-.js-focus-visible .crayons-btn.focus-visible:focus-within {
+.js-focus-visible .crayons-btn.focus-visible:focus-within,
+label[class*='crayons-btn']:focus-within // label for input[type="file"] made to look like a button as the browse... button can't be styled.
```

Comment:
This also fixes https://github.com/forem/forem/issues/15234 🎉

Reply:
I'm going to put this in a separate PR to unblock that issue, and it's really a global fix.